src/v2/Apps/Collect/Routes/Collection/Components/Header/index.tsx

```diff
@@ -103,18 +103,20 @@ export const featuredArtistsEntityCollection: (
               onOpenAuthModal={() => handleOpenAuth(mediator, artist)}
               render={({ is_followed }) => {
                 return (
-                  <Sans
-                    size="2"
-                    weight="medium"
-                    color="black"
-                    data-test="followArtistButton"
-                    style={{
-                      cursor: "pointer",
-                      textDecoration: "underline",
-                    }}
-                  >
-                    {is_followed ? "Following" : "Follow"}
-                  </Sans>
+                  <Clickable>
```

Comment:
So it's not immediately clear to me: you can tab focus this and `enter`/`space` executes the follow? I only ask because the `onClick` handler is above it so I guess it bubbles?

Reply:
Oh, I see, it gets wrapped in that span (with the tabIndex/onClick handler etc). In that case you don't need to use `Clickable` at all since it's just custom rendering the interior children.

Reply:
So we still need `Clickable` in order to get the purple highlights, unless I'm missing something and we can/should easily add that to the `span` implementation below.

Reply:
That shouldn't be needed really — just having a `tabindex="0"` allows it to be keyboard focusable: [example](https://jsfiddle.net/dzucconi/aL0zvjru/1/) — but if it works!

Reply:
I didn't dig deeper, but when removing `Clickable` the purple box went away 😭... Maybe because of the button-within-a-button thing?

Reply:
> Maybe because of the button-within-a-button thing?

So long as it actually triggers the follow, which it should since it'll bubble up: https://jsfiddle.net/dzucconi/aL0zvjru/4/ definitely sort of weird to see how this works if you focus the inner button.

Reply:
It does trigger the follow but it's a bad experience IMO to not have the highlight. 🤷‍♀️ 

I think this is good-to-go but if you want me to remove the `Clickable`s I can.